packages/docs/src/examples/va-badge/WithCard.vue

```diff
@@ -3,7 +3,7 @@
     <va-badge text="New" overlap>
       <va-card>
         <va-card-content>
-          Next update is right behind the corner
```

Comment:
That's just generic text, you can add universal translation.

Reply:
moved the sentence to a generic text with universal translation